assets/src/dashboard/components/cardGridItem/cardPreview.js

```diff
@@ -224,6 +211,7 @@ const CardPreviewContainer = ({
               tabIndex={tabIndex}
               data-testid="card-center-action"
               type={BUTTON_TYPES.SECONDARY}
+              size={BUTTON_SIZES.SMALL}
               {...getActionAttributes(centerAction.targetAction)}
```

Comment:
Should this be a `Link` or do we want to keep it as a button? I know we're passing `as="a"` to it now 🤔

Reply:
There's room in the `Button` component for links styled as buttons so it should be ok. The center action could be a link or a button depending on context.